client/src/layouts/LoginLayout.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { Button, Divider, Form, Grid, Header, Image, Segment } from 'semantic-ui-react'
+
+const gridStyle = { height: '100%' };
+const colStyle = { maxWidth: 450 };
+
+export default class LoginPage extends React.Component {
+  state = { email: '', password: '' };
+
+  handleChange = (_, { name, value }) => this.setState({ [name]: value });
+
+  onLogin = () => {
+    const { email, password } = this.state;
+    this.props.onSubmitLogin(email, password);
+  }
+
+  onSignup = (e) => {
+    e.preventDefault();
+    const { email, password } = this.state;
+    this.props.onSubmitSignup(email, password);
+  }
+
+  render () {
+    return (
+      <div className='login-form'>
+        <Grid textAlign='center' style={gridStyle} verticalAlign='middle'>
+          <Grid.Column style={colStyle}>
+            <Header as='h2' color='teal' textAlign='center'>
+              <Image src='/favicon.ico' /> Login to your account
+            </Header>
+            <Form size='large' onSubmit={this.onLogin}>
```

Comment:
For all react form-related things, I highly highly highly recommend Formik. It'll handle sooo many form use cases for you:  you don't need to manually write the handlers (issue for if you have non-traditional input elements like checkboxes, or for nesting); code looks a lot cleaner; nesting is super easy; you can see exactly what the schema of your form is (it's still state-based)

And even if you use formik, you still can style it however you want - it's just a backing framework to handle forms in React.

https://www.youtube.com/watch?v=oiNtnehlaTo <-- this video is all you'll need to watch and you'll be a master in Formik